space-plugins/story-starter/starters/nuxt/server/api/stories.ts

```diff
@@ -32,60 +21,31 @@ export default defineEventHandler(async (event): Promise<StoriesResponse> => {
 		}),
 	});
 
-	const response = await storyblokFetch(accessToken).getStories({
-		spaceId,
-		page: query.page,
-		perPage: query.perPage,
-		slug: query.slug,
-		query: query.query,
-	});
-
-	return response;
-});
-
-const storyblokFetch = (accessToken: string) => {
-	const defaults = {
-		perPage: 25,
-		page: 1,
-		version: 'published',
-	};
-
 	// This is a management API instance because of the `oauthToken`.
 	const storyblokClient = new StoryblokClient({
 		oauthToken: `bearer ${accessToken}`,
 	});
 
-	const getStories: GetStories = async ({
-		spaceId,
-		perPage,
-		page,
-		version,
-		slug,
-		query,
-	}) => {
-		const params: ISbStoriesParams = {
-			version: version ?? (defaults.version as Version),
-			per_page: perPage ?? defaults.perPage,
-			page: page ?? defaults.page,
-		};
-		if (slug) {
-			params.by_slugs = slug;
-		}
-		if (query) {
-			// @ts-expect-error there is a typing error from the library
-			params.text_search = query;
-		}
-		const { data, total } = await storyblokClient.get(
-			`spaces/${spaceId}/stories`,
-			params
-		);
-
-		return {
-			stories: data.stories as Story[],
-			perPage: perPage || data.per_page,
-			total,
-		};
+	const params: ISbStoriesParams = {
+		version: 'published',
```

Comment:
In the future, we could accept the `version` to be passed when calling the `useStories`, wdyt? 🤔 

May allowing it to fetch also a draft can be relevant for some space plugins, IDK.